docs/REST API/rest-api.md

```diff
@@ -2524,6 +2524,16 @@ Returns the information of the specified host.
         "response": "",
```

Comment:
I think we leave "pending" as `""` just in case anyone's relying on the API returning that value for "no response" @noahtalerman wdyt?